easybuild/easyconfigs/d/double-conversion/double-conversion-3.1.4-GCCcore-8.3.0.eb

```
@@ -14,15 +14,24 @@ checksums = ['95004b65e43fefc6100f337a25da27bb99b9ef8d4071a36a33b5e83eb1f82021']
 
 builddependencies = [
     ('binutils', '2.32'),
-    ('SCons', '3.1.1'),
+    ('CMake', '3.15.3'),
 ]
 
-installopts = "DESTDIR=%(installdir)s prefix='' && "
-installopts += "mkdir %(installdir)s/include && cp double-conversion/*.h %(installdir)s/include"
+separate_build_dir = True
+
+local_buildtype = '-DCMAKE_BUILD_TYPE=Release '
+configopts = [
+    local_buildtype + c for c in (
+        '',
+        '-DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_STATIC_LIBRARY_SUFFIX_CXX=_pic.a',
+        '-DBUILD_SHARED_LIBS=ON'
+    )
+]
```

I'm all in for avoiding repetition, but in this case I would just use a pure list, it is shorter an easier to read. Additionally, could you add a comment explaining what is each compilation doing? I see that last one adds shared libs, but what is the different between first and second one?
```suggestion
configopts = [
    '-DCMAKE_BUILD_TYPE=Release',
    '-DCMAKE_BUILD_TYPE=Release -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_STATIC_LIBRARY_SUFFIX_CXX=_pic.a',
    '-DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON'
]
```

---

I agree. Actually there is a better way now using `build_type` which wasn't available when I made this. Changed here: https://github.com/easybuilders/easybuild-easyconfigs/pull/9659/commits/5b23e0b6578cf140422f45f29206eba4ee0f3ab9

> I see that last one adds shared libs, but what is the different between first and second one?

I added a comment. The difference is that the first builds a shared library without -fPIC. There seems to be use for a `libdouble-conversion.a` and `libdouble-conversion_pic.a` and I don't want to break this (done by the original Scons too)